scripts/apps/desks/views/desk-config-modal.html

```diff
@@ -183,6 +183,10 @@ <h3 class="modal__heading" ng-show="desk.edit._id" translate>Edit "{{desk.edit.n
                             </div>
                         </div>
                     </div>
+                    <div class= "form__row" >
+                        <span sd-switch ng-model="desk.edit.restriction"></span>
```

Comment:
Improve naming, `desk.restriction` doesn't tell what is restricted. Better use `desk.send_to_desk_allowed` that would default to `true` unless toggled off.